hypervisor/boot/reloc.c

```diff
@@ -104,7 +104,7 @@ void _relocate(void)
 	 * Need to subtract the relocation delta to get the correct
 	 * absolute addresses
 	 */
-	trampoline_end = (uint64_t)_ld_trampoline_end - delta;
+	trampoline_end = (uint64_t)&_ld_trampoline_end - delta;
```

Comment:
trampoline_end = (uint64_t)(&_ld_trampoline_end) - delta;  
to make it consistent and easy to understand